tasks/sep/006-0-sc-guardian-updates/.env

```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL=https://1rpc.io/sepolia	 # Fill this in with an L1 Sepolia RPC URL that has archive data access.
+OWNER_SAFE=0xa87675ebb9501C7baE8570a431c108C1577478Fa
```

Comment:
2 of 10 Gnosis Safe https://sepolia.etherscan.io/address/0xa87675ebb9501C7baE8570a431c108C1577478Fa#readProxyContract

How can I verify that this is the expected safe?

Reply:
It's a brand new Safe, so I think the key thing is to verify that its config is as expected. We will also need to figure out the right way to track it in the superchain registry, but that's not a blocker here.

```suggestion
# See sepolia.json for config values being verified:
# https://github.com/ethereum-optimism/optimism/pull/10224/files
OWNER_SAFE=0xa87675ebb9501C7baE8570a431c108C1577478Fa
```

Reply:
looks like that safe has 9 owners not 10, is that okay?